src/cookbooks/clients_last_seen_bits.md

```diff
@@ -0,0 +1,136 @@
+DAU generally looks like:
+
+```
+SELECT
+  COUNT(days_since_seen < 28) AS mau,
+  COUNT(days_since_seen <  7) AS wau,
+  COUNT(days_since_seen <  1) AS dau
+FROM
+  telemetry.clients_last_seen
+```
+
+Note that this particular client is about to fall outside the WAU window.
+If the client doesn't send a main ping on 2020-01-29, the new `days_seen_bits`
+pattern will look like:
+
+```
+    0  0  0  0  0  0  0  0  0  0  0  0  0  1  0  0  0  0  0  0  1  0  0  0  0  0  0  0
+    ──────────────────────────────────────────────────────────────────────────────────
+    │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │
+    │ 26  │ 24  │ 22  │ 20  │ 18  │ 16  │ 14  │ 12  │ 10  │  8  │  6  │  4  │  2  │  0
+   27    25    23    21    19    17    15    13    11     9     7     5     3     1
+   
+  └──────────────────────────────────────────────────────────────────────────────────┘
+  MAU                                                             └──────────────────┘
+                                                                   WAU             └─┘
+                                                                                   DAU
+```
+
+The `days_seen_bits` value is now `16512` and the user is indeed outside the WAU window:
+
+```
+SELECT udf.bits28_days_since_seen(16512)
+-- 7
+```
+
+## Retention: Forward-looking windows
+
+For retention calculations, we user forward-looking windows. This means that
+when we report a retention value for 2020-01-01, we're talking about what
+portion of clients active on 2020-01-01 are still active some number of days
+later.
+
+When we were talking about backward-looking windows, our anchor date or "day 0"
+was always the `submission_date`. When we define forward-looking windows, however,
+we always choose an anchor date some time in the past. How we number the individual
+bits depends on what anchor date we choose.
+
+In GUD, we show a "Week 1 Retention" which considers a window of 14 days.
+
+Let's consider the same user from before, grabbing the `days_seen_bits` value
+with `submission_date = 2020-01-28`. We'll still choose a window that _ends_
+on the submission date, but the anchor date (day 0) will now be 13 days earlier:
+
+
+```
+    0  0  0  0  0  0  0  0  0  0  0  0  0  0  1  0  0  0  0  0  0  1  0  0  0  0  0  0
+    ──────────────────────────────────────────────────────────────────────────────────
+                                              │  │  │  │  │  │  │  │  │  │  │  │  │  │
+                                              │  1  │  3  │  5  │  7  │  9  │ 11  │ 13
+                                              0     2     4     6     8    10    12                                     
+                                             └───────────────────┘
+                                                    Week 0       └───────────────────┘
+                                                                        Week 1
+```
+
```

Comment:
I think some kind of explanation of what we just determined by this calculation would be helpful here, instead of relying on the reader to fill in the blanks.

Reply:
Will do.